agile-development/team-agreements/team-manifesto/readme.md

```diff
@@ -0,0 +1,78 @@
+# Team Manifesto
+
+## Introduction
+
+CSE teams work with a new development team in each customer engagement which requires a phase of introduction & knowledge transfer before starting an engagement.
+
+Completion of this phase of ice-breakers and discussions around the standards takes time however also required to start increasing the learning curve of the new team.
+
+A team manifesto is a soft contract among team members which summarizes the basic principles and values of the team and aiming to provide a consensus about
+expectations from each team member in order to deliver high quality output at the end of each engagement.
+
+It aims to reduce the time on setting the right expectations and providing a consensus among team members as well as clarifying the question of "How does the new team develop the software?"
```

Comment:
I don't understand this sentence, can you clarify a bit what you mean here?

Reply:
kindly check if it's clear now in 39192e09c809ea42f7c2506041b7348d73761940 ? thanks :)